### DISCUSSION TITLE
Suggestion: Configured Feature Preview

### DISCUSSION THREAD

**Discussion by Erkko68** [syldium/worldgen#5] [Ideas]
It would be great to have some kind of image preview when you create a structure in the "Configured Feature" section. This will help make it more intuitive and easy to create a new structure.

**Comment by syldium** [comment]
I totally agree, a visualization would be useful. The main problem is that the generation code has a copyright and the creation of a (3d?) viewer takes time. I take this into consideration nevertheless, maybe for the update with 1.17?

**Reply by Erkko68** [reply]
Don't worry if you cannot implement it, it is still a very useful tool, and keep going with this great project! 😄
